lib/index.js

```diff
@@ -84,6 +86,8 @@ const configs = {
         { files: ["**/*.mjs"], plugins: { n: base }, ...esmConfig.flat },
         { files: ["**/*.cjs"], plugins: { n: base }, ...cjsConfig.flat },
     ],
+    // @ts-ignore -- tsc warns incompatible languageOptions, but it's fine
```

Comment:
This feels strange to have to suppress -- is it because `lib/configs/all.js` is missing the `@type {import('eslint').Linter.FlatConfig}` at the top of the config?